src/profile-logic/call-tree.js

```diff
@@ -701,15 +700,15 @@ export function computeCallTreeTimingsInverted(
   const totalPerRootNode = new Map();
   const rootNodesWithChildren = new Set();
   const seenRoots = new Set();
-  for (let i = 0; i < callNodeLeaf.length; i++) {
-    const leaf = callNodeLeaf[i];
-    if (leaf === 0) {
+  for (let i = 0; i < callNodeSelf.length; i++) {
+    const self = callNodeSelf[i];
+    if (self === 0) {
       continue;
     }
 
     // Map the non-inverted call node to its corresponding root in the inverted
     // call tree. This is done by finding the inverted root which corresponds to
-    // the leaf function of the non-inverted call node.
+    // the self function of the non-inverted call node.
```

Comment:
Hmm, maybe I don't understand what you mean by leaf. I consider "self" to mean "end of the stack, function in which the time was spent". And I consider "leaf" to mean "a node which doesn't have any children". And this code previously also used "leaf time" to mean:
 - in the non-inverted tree,: "the same as self time"
 - in the inverted tree: "total time of this node minus sum of the total times of the child nodes, but not called self time because self time is in the roots" 

In this comment we are talking about a non-inverted call node.